### PR TITLE
(Skill) Add knock knock joke

### PR DESCRIPTION
**Pull request by tjcramer**
If your PR is related to a contribution to the taxonomy, please, fill
out the following questionnaire. If not, replace this whole text and the
following questionnaire with whatever information is applicable to your PR.


**Describe the contribution to the taxonomy**

<!-- A concise description of what the contribution brings, replace "..." in the bullet list -->

- Training the model to do knock knock jokes by giving 20 examples


**Input given at the prompt**

<!-- What you entered, replace "..." -->

```
   tell me a knock knock joke
```


**Response from the original model**


<!-- What you received from the original model in response to your input, 
replace "..." -->

```
>>> tell me a knock knock joke                                                                                                                                                                  [S][default]
╭──────────────────────────────────────────────────────────────────────────────────── models/merlinite-7b-Q4_K_M.gguf ─────────────────────────────────────────────────────────────────────────────────────╮
│ Sure, here's a classic one for you:                                                                                                                                                                                                                                                                                                                                                 │
│ Knock, knock.                                                                                                                                                                                            │
│ Who's there?                                                                                                                                                                                             │
│ Lion.                                                                                                                                                                                                    │
│ Lion who?                                                                                                                                                                                                │
│ Lion I tell you, I saw a lion in the zoo today!                                                                                                                                                          │
│                                                                                                                                                                                                          │
│ I'm glad I could bring some joy into your day! If you have any other requests or questions, please don't hesitate to ask. I'm here to help!                                                              │
╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────── elapsed 3.435 seconds ─╯
>>> tell me a knock knock joke                                                                                                                                                                  [S][default]
╭──────────────────────────────────────────────────────────────────────────────────── models/merlinite-7b-Q4_K_M.gguf ─────────────────────────────────────────────────────────────────────────────────────╮
│ I'd be happy to share another joke with you:                                                                                                                                                             │
│ Knock, knock.                                                                                                                                                                                            │
│ Who's there?                                                                                                                                                                                             │
│ Interior.                                                                                                                                                                                                │
│ Interior who?                                                                                                                                                                                            │
│ Interior decorate your answer with a little more creativity!                                                                                                                                             │
│                                                                                                                                                                                                          │
│ I hope this made you laugh! If you have any other requests or questions, please feel free to ask. I'm here to make your day a little brighter!                                                           │
╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────── elapsed 3.324 seconds ─╯
>>> tell me a knock knock joke                                                                                                                                                                  [S][default]
╭──────────────────────────────────────────────────────────────────────────────────── models/merlinite-7b-Q4_K_M.gguf ─────────────────────────────────────────────────────────────────────────────────────╮
│ Absolutely! Here's another one:                                                                                                                                                                          │
│ Knock, knock.                                                                                                                                                                                            │
│ Who's there?                                                                                                                                                                                             │
│ Bike.                                                                                                                                                                                                    │
│ Bike who?                                                                                                                                                                                                │
│ Bike I told you, I saw a bike shed near the park!                                                                                                                                                        │
│                                                                                                                                                                                                          │
│ I hope this brought a smile to your face! If you have any other requests or questions, please don't hesitate to ask. I'm here to help!                                                                   │```


**Response from the fine-tuned model**


<!-- Generate a synthetic dataset based on your newly added seed data; train the model 
with the synthetic data and now re-test the model's response with the same prompt.
Replace "..." with what you receive with the finetuned model. -->

```
>> tell me a knock knock joke                                                                                                                                                                  [S][default]
╭────────────────────────────────────────────────────────────────────────── ibm-merlinite-7b-mlx-q-fused-pt/ggml-model-f16.gguf ───────────────────────────────────────────────────────────────────────────╮
│ (j) Knock knock!                                                                                                                                                                                         │
│ (b) Who's there?                                                                                                                                                                                         │
│ (j) Lettuce!                                                                                                                                                                                             │
│ (b) Lettuce who?                                                                                                                                                                                         │
│ (j) Lettuce in, it's too cold outside!                                                                                                                                                                   │
╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────── elapsed 5.658 seconds ─╯
>>> tell me a different knock knock joke                                                                                                                                                        [S][default]
╭────────────────────────────────────────────────────────────────────────── ibm-merlinite-7b-mlx-q-fused-pt/ggml-model-f16.gguf ───────────────────────────────────────────────────────────────────────────╮
│ (g) Knock knock!                                                                                                                                                                                         │
│ (b) Who's there?                                                                                                                                                                                         │
│ (g) Farmer McMoo!                                                                                                                                                                                        │
│ (b) Farmer McMoo who?                                                                                                                                                                                    │
│ (g) Farmer McMoo, because I need to ask you: "Why does my cow have so many problems with digestion?"                                                                                                     │
│ (b) Because he's eating too much hay!                                                                                                                                                                    │
╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────── elapsed 9.226 seconds ─╯
>>> tell me a different knock knock joke                                                                                                                                                        [S][default]
╭────────────────────────────────────────────────────────────────────────── ibm-merlinite-7b-mlx-q-fused-pt/ggml-model-f16.gguf ───────────────────────────────────────────────────────────────────────────╮
│ (g) Knock knock!                                                                                                                                                                                         │
│ (b) Who's there?                                                                                                                                                                                         │
│ (g) Boo!                                                                                                                                                                                                 │
│ (b) Boo who?                                                                                                                                                                                             │
│ (g) Boo, because I want to ask you: "Why did the ghost go to the haunted house?"                                                                                                                         │
│ (b) To get the boo-ty! (pun intended)                                                                                                                                                                    │
╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────── elapsed 8.563 seconds ─╯
```

**Contribution checklist**

<!-- Insert an x between the empty brackets: [ ] >> [x] -->

- [X ] The contribution was tested with `lab generate`
- [X ] No errors or warnings were produced by `lab generate`
- [X] All [commits are signed off](https://github.com/instruct-lab/taxonomy/blob/main/CONTRIBUTING.md#legal) (DCO)
- [X] The `qna.yaml` file contains at least 5 `seed_examples`
- [ ] The `qna.yaml` file was [linted](https://yamllint.com) and [prettified](https://onlineyamltools.com/prettify-yaml) ([yaml-validator](https://jsonformatter.org/yaml-validator) can do both)

### CODE REVIEW

**Review by mairin — APPROVED** [review]
LGTM
